src/server-code/admin-server.js

```diff
@@ -219,6 +221,36 @@ export default function runAdminServer(serverFalcorModel) {
     });
   }
 
+  const CLIENT_ID = '870681894101-ce7tdp6h9fvt2jfrqaaalim1s2n48ie3.apps.googleusercontent.com';
+  const WHITELIST = ['xt405@nyu.edu', 'tjk343@nyu.edu', 'ks3583@nyu.edu', 'kw1553@nyu.edu', 'hct245@nyu.edu',
+        'zmm228@nyu.edu', 'ego225@nyu.edu']; // eslint-disable-line max-len
+
+  app.post('/googlelogin', (req, res) => {
+    const token = req.body.data;
+    const endpoint = `https://www.googleapis.com/oauth2/v3/tokeninfo?id_token=${token}`;
+
+    request.get(endpoint, (error, response, body) => {
+      if (error) {
+        res.status(500).send('google login error');
```

Comment:
Maybe make this a bit more descriptive. Such as "Internal server error while authenticating Google token" or feel free to come up with something better